composer.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "fireworkweb/smpte",
+    "description": "Easily deal with Timecode SMPTE format in PHP",
+    "keywords": ["timecode", "smpte", "subtitle"],
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Firework Web & Mobile",
+            "email": "contact@fireworkweb.com"
+        }
+    ],
+    "type": "project",
+    "require": {
+        "php": "^7.0"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^6"
+    },
+    "autoload": {
+        "psr-4": {
+            "FireworkWeb\\SMPTE\\": "src/",
+            "FireworkWeb\\Tests\\": "tests/"
```

Comment:
remove tests to `autoload-dev`:

https://github.com/laravel/framework/blob/5.7/composer.json#L101-L108